netty/src/main/java/io/grpc/transport/netty/NettyChannelBuilder.java

```diff
@@ -70,6 +73,14 @@ private NettyChannelBuilder(SocketAddress serverAddress) {
   }
 
   /**
+   * Specify the channel type to use, by default we use {@link NioSocketChannel}.
+   */
+  public NettyChannelBuilder channelType(Class<? extends Channel> channelType) {
+    this.channelType = channelType;
```

Comment:
checkNotNull? I know it will be found later, but here the backtrace is more useful.